sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -318,12 +327,42 @@ template <typename... Args>
 consteval explicit device_global(Args&&... args);
 ----
 |
+Available only if sizeof...(Args) != 1 or the one argument in args is not a device_global.
+
 Constructs a `device_global` object, and implicit storage for `T` in the global address space on each device that may access it.
 
 The object of type `T` is initialized from the `args` parameter pack using list initialization as defined in the {cpp} specification.
 
 `T` must be trivially destructible.
 
+// --- ROW BREAK ---
+a|
+[source,c++]
+----
+constexpr device_global(const device_global &other);
+----
+|
+Available if `PropertyListT::has_property<device_image_scope_key>() == false`.
+
+Constructs a `device_global` object, and implicit storage for `T` in the global address space on each device that may access it.
+
+The storage on each device for `T` is initialized with a copy of the storage in `other`.
+
+// --- ROW BREAK ---
+a|
+[source,c++]
+----
+template <typename OtherT, typename OtherProps>
+constexpr device_global(const device_global<OtherT, OtherProps> &other);
+----
+|
+Available if `PropertyListT::has_property<device_image_scope_key>() == false` and
+`std::is_convertible_v<OtherT, T> == true`;
+
+Constructs a `device_global` object, and implicit storage for `T` in the global address space on each device that may access it.
+
+The storage on each device for `T` is initialized with a copy of the initial value of `other`. The behavior is undefined if `other` has been written to prior to a call this constructor.
```

Comment:
It seems strange that the sentence "The behavior is undefined if ..." does not also exist for the non-templated overload above.  I assume we have the same restriction there?

Reply:
Ah, you are right. The ctor was added back after this behavioral note caveat was added. Good catch.